src/js/components/CosmosErrorMessage.js

```diff
@@ -71,7 +113,10 @@ CosmosErrorMessage.propTypes = {
   className: React.PropTypes.string,
   error: React.PropTypes.shape({
     message: React.PropTypes.string,
-    type: React.PropTypes.string
+    type: React.PropTypes.string,
+    data: React.PropTypes.shape({
+      errors: React.PropTypes.array
```

Comment:
You are right! Changing `data` from `shape` to `object` in order to be more generic.